### PR TITLE
added comicbrise

### PR DESCRIPTION
**Pull request by gammesB**
added comicbrise
example: https://comic-brise.com/contents/oshiai/

### CODE REVIEW

File: src/web/mjs/connectors/ComicBrise.mjs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '.post-title');
+        let id = uri.pathname;
+        let title = data[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const msg = 'This website does not support mangas/chapters, please copy and paste the links containing the chapters directly from your browser into HakuNeko.';
```
Comment (27136773) [inline src/web/mjs/connectors/ComicBrise.mjs]: It seems website provide titles, why not support them?
https://comic-brise.com/titlelist/

File: src/web/mjs/connectors/ComicBrise.mjs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        throw new Error(msg);
+    }
+
+    async _getChapters(manga) {
+        const uri = new URL(manga.id, this.url);
+        const request = new Request(uri, this.requestOptions);
+		const dom = await this.fetchDOM(request);
```
Comment (27136773) [inline src/web/mjs/connectors/ComicBrise.mjs]: Prevent mixing spaces and tabs for indentation.

File: src/web/mjs/connectors/ComicBrise.mjs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        const chaptersNames = dom.querySelectorAll('.modal.modal-chapter .modal-body .primary-title');
+        const chaptersLink = dom.querySelectorAll('.modal.modal-chapter .modal-body .banner-trial a');
+        const chaptersFree = dom.querySelectorAll('.modal.modal-chapter .modal-body .banner-trial source');
```
Comment (27136773) [inline src/web/mjs/connectors/ComicBrise.mjs]: Consider to query against `.modal.modal-chapter .modal-body` and then map the list by query each element itself,
e.g.
```javascript
const elements = await this.fetchDOM(request, '.modal.modal-chapter .modal-body');
return elements.map(element => {
    const id = element.querySelector('.banner-trial a').pathname;
    const title = element.querySelector('.primary-title').textContent.trim();
    // ...
});
```

Comment (27136773) [reply src/web/mjs/connectors/ComicBrise.mjs]: May utilize the `filter` function to exclude non-free titles

**Comment by Sheepux** [comment]
Make sure to close the "Resolve conversation" if you've solved the issue and also click the re-request review to let us know the PR is read for review again :)

**Comment by gammesB** [comment]
O sorry I didn't know you had to do that after submitting changes. Will do it straight away!

**Review by 27136773 — APPROVED** [review]

**Pull request merged**